introduction_to_applying_machine_learning/identify_key_insights_from_textual_document/document_summarization.ipynb

```diff
@@ -0,0 +1,522 @@
+{
```

Comment:
<p> </p><p>Use present tense - we'll be using should be we use and more</p><p> </p><p> </p><p>Looks like there is a lot to be converted to present tense - can you convert for the rest of the notebooks, thank you!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>